src/Symfony/Component/Cache/Traits/PhpArrayTrait.php

```diff
@@ -102,7 +102,7 @@ public function warmUp(array $values)
 
         $tmpFile = uniqid($this->file, true);
 
-        file_put_contents($tmpFile, $dump);
+        file_put_contents($tmpFile, $dump, LOCK_EX);
```

Comment:
This one is wrong: the file has a random name exactly to prevent the need for a lock.

Reply:
Oh, new false-positive. I'll revert this one.